src/frontend/src/NavBar/components/UserMenu/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, createStyles } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { AppThemeOptions, Menu, MenuItem } from "czifui";
+import React from "react";
+import { Link } from "react-router-dom";
+import { API } from "src/common/api";
+import { ROUTES } from "src/common/routes";
+
+const useStyles = makeStyles((theme: AppThemeOptions) => {
+  const palette = theme.palette;
+
+  return createStyles({
+    text: {
+      color: palette?.common?.white,
+    },
+  });
+});
+
+interface UserMenuProps {
+  user: string;
+}
+
+const UserMenu = ({ user }: UserMenuProps): JSX.Element => {
+  const [anchorEl, setAnchorEl] = React.useState<Element | null>(null);
+
+  const classes = useStyles();
+
+  const handleClick: React.MouseEventHandler<HTMLButtonElement> = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
```

Comment:
do we need this empty statement here?

Reply:
Great question! This `<></>` in React means a [Fragment](https://reactjs.org/docs/fragments.html), which allows us side step the React rule of always returning a tree with a single root element

>Fragments
A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.